src/core/bytecodedump.c

```diff
@@ -176,7 +176,7 @@ char * MVM_bytecode_dump(MVMThreadContext *tc, MVMCompUnit *cu) {
     MVMuint32 lineloc;
     MVMuint16 op_num;
     const MVMOpInfo *op_info;
-    MVMuint32 operand_size;
```

Comment:
True.  However, static analysis (via Coverity scan) showed that `operand_size` could still be used uninitialised.  What is required is an unambiguous way to ensure that `operand_size` won't be used without being set, now, or some time in the future.